### ISSUE TITLE
Suggestion: consistency across Nextcloud action naming

### ISSUE THREAD

**Issue by leoossa** [nextcloud/deck#3240]
I'd like to suggest keeping consistency across action naming. 
In Deck, it's often to have `Edit` action, but for example, comment has named `Update` action for editing comment. 

### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by stefan-niedermann** [comment]
I agree, though the issue description is on such a high level that it is nearly impossible to solve. I suppose to gather more concrete points like your sample so we can define a goal where this issue can be marked as "solved" eventually.

Regarding your sample: In my opinion it should be `Edit` in both cases, though i am not sure whether this needs to be fixed in upstream vue components (this can cause other inconsistencies in other places where they are used) or within the Deck app itself (this would be quite easy then).

**Comment by juliushaertl** [comment]
Most likely to be fixed in the deck app itself. I remember talking about action naming in the design guidelines discussion with @nimisha-vijay so maybe we can do that once those have been settled. I personally think that an action should also have an objective that it refers to so I would always try to use something like "Edit card" or "Edit comment".
